recipes/rmblast/meta.yaml

```diff
@@ -1,16 +1,16 @@
 {% set name = "rmblast" %}
-{% set version = "2.9.0" %}
+{% set version = "2.10.0" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
   url: https://ftp.ncbi.nlm.nih.gov/blast/executables/blast+/{{ version }}/ncbi-blast-{{ version }}+-src.tar.gz
-  sha256: a390cc2d7a09422759fc178db84de9def822cbe485916bbb2ec0d215dacdc257
+  sha256: 3acdd9cec01c4f43e56aeaf89049cb8f8013d60b9c1705eced10166967f1d926
   #patches:
```

Comment:
patches are not used anymore, isn't it? So we can remove them?

Reply:
I thought the "Needs the -p1 option, applied in build.sh" comment was helpful in providing the rationale for why the patch is applied in build.sh, but it's fine to remove the (commented-out) patches field entirely from meta.yaml if that's preferred.